src/classes/PMT_PaymentCreator.cls

```diff
@@ -32,7 +32,7 @@
 * @date 2011 (2.1)
 * @description Controls the creation of payment objects from Opportunity trigger
 */
-public without sharing class PMT_PaymentCreator {
+public with sharing class PMT_PaymentCreator {
 
```

Comment:
since this class is used by the trigger, shouldn't it be empty to inherit the trigger's without sharing?